hack/pluto.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# wrapper to call pluto
+
+set -e
+
+source hack/sh/rc.sh
+source hack/sh/deps/pluto.sh
+
+helm template $1 | pluto detect -
```

Comment:
ok, so i realised why there was no `err`.

we should do something like this to handle errors:
```suggestion
source hack/sh/_functions.sh
source hack/sh/deps/pluto.sh

chart_dir=$1

[[ -d $chart_dir ]] || err "No chart dir given. Usage: hack/pluto.sh chart/name"

helm template $chart_dir | pluto detect -
```